subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/matchers/UserAgentMatcher.java

```diff
@@ -49,6 +52,6 @@ private String expectedUserAgentString() {
         String osName = System.getProperty("os.name");
         String osVersion = System.getProperty("os.version");
         String osArch = System.getProperty("os.arch");
-        return String.format("%s/%s (%s;%s;%s) (%s;%s;%s)", applicationName, version, osName, osVersion, osArch, javaVendor, javaVersion, javaVendorVersion);
```

Comment:
We need to keep the same type of format here in order to keep our analytics working. 

I'd consider it a nice favor if you wrote `"%s/0 (%s;%s;%s) (%s;%s;%s)" keeping "/0" there

Reply:
This version isn't parsed from the `User-Agent` string. It's parsed from the download file name for analytics, but it'd be nice if we don't have to change up the analytics code.

Reply:
Changed in 00862b8160a.